domains/tiuri.json

```diff
@@ -7,6 +7,6 @@
   },
   "record": {
     "A": [
-      "138.201.76.10"
+      "212.227.129.140"
```

Comment:
Please upload your website to the server.

Reply:
I did, and I changed from server. You cant see the website when you visit the ip because it works with nginx virtualhosts.

Reply:
But I Will bring it online on a different domain So you can see the website

Reply:
@ituir Please send the domain

Reply:
Is this it?
![Screenshot_20221121_153524_Chrome](https://user-images.githubusercontent.com/87287585/202991489-fd9fd8f0-f01a-4fc1-83a6-6fe6fb5d4527.jpg)

Reply:
yes, it is not very complicated!

Reply:
> yes, it is not very complicated!

Sorry, you can't use that as your website.